numpy/core/src/umath/loops.c.src

```diff
@@ -1558,6 +1558,15 @@ FLOAT_@func@(char **args, npy_intp const *dimensions, npy_intp const *steps, voi
 
 /**end repeat**/
 
+NPY_NO_EXPORT NPY_GCC_OPT_3 void
+DOUBLE_exp(char **args, npy_intp *dimensions, npy_intp *steps, void *NPY_UNUSED(data))
```

Comment:
These declarations are different from the others, causing a compiler warning: the `const` is missing

Reply:
Thx, fixed it.